pkg/ingress/model_build_managed_sg.go

```diff
@@ -2,7 +2,7 @@ package ingress
 
 import (
 	"context"
-	"crypto/md5"
+	"crypto/sha1"
```

Comment:
if md5 is not acceptable, sha1 would also not be. lets replace with sha256

Reply:
updated with sha256 hashes :D